src/components/picker/types.tsx

```diff
@@ -220,11 +307,11 @@ export interface PickerItemProps extends Pick<TouchableOpacityProps, 'customValu
   /**
    * Custom function for the item label (e.g (value) => customLabel)
    */
-  getItemLabel?: PickerProps['getItemLabel'];
+  getItemLabel?: PickerPropsDeprecation['getItemLabel'];
```

Comment:
If this prop is not deprecated why its type is on the PickerPropsDeprecation?

Reply:
I'll add this new type in separate PR I have a task about it.

Reply:
Please merge master, I've merged this PR